README.md

```diff
@@ -61,6 +61,30 @@ Settings are found in `settings.py`
     export CLICKHOUSE_SERVER=127.0.0.1:9000
 
     make test
+    
+## Testing Against Sentry
+
+```
+workon snuba
+git checkout your-snuba-branch
+snuba api
+```
+And then in another terminal 
+```
+workon sentry
+git checkout master
+git pull
+sentry devservices up --exclude=snuba
+```
+This will get the most recent version of Sentry on master, and bring up all snuba's dependencies. 
+
+You will want to run the following Sentry tests:
+```
+USE_SNUBA=1 make test-acceptance
+USE_SNUBA=1 make test-snuba
+make test-python
```

Comment:
It looks like  `USE_SNUBA` is used more extensively than I initially assumed. See: https://github.com/getsentry/sentry/blob/234449f7232331d5584856553669aaf113aa6e8b/src/sentry/utils/pytest/sentry.py#L115 I need to rephrase this. Is it valuable to include a note about the python tests if snuba is not currently used? Currently it looks like just the legacy backend is working.

Reply:
We could probably remove `USE_SNUBA` and make its behavior the default now that most folks are using it and it is available in acceptance tests.

Reply:
We can't just yet, but the work I'm doing in https://github.com/getsentry/sentry/pull/13178 will make it easier